demo-shell/src/app/components/app-layout/app-layout.component.html

```diff
@@ -73,15 +71,16 @@
 </adf-sidenav-layout>
 <app-log></app-log>
 
-<mat-menu #nestedMenu="matMenu" [xPosition]="'before'">
+<mat-menu #nestedMenu="matMenu" xPosition="after" class="nestedMenu">
```

Comment:
Is the nestedMenu one of ADF's classes? if so, could we make it follow the rules with the prefix `adf-` ?

Reply:
its just for demoshell

Reply:
ok then, although i strongly suggest that we use that prefix everywhere. But been only demoshell is ok